identity-wallet/src/state/reducers/authorization.rs

```diff
@@ -67,6 +68,19 @@ pub async fn read_authorization_request(state: AppState, action: Action) -> Resu
             info!("client_name in credential_offer: {:?}", client_name);
             info!("logo_uri in read_authorization_request: {:?}", logo_uri);
 
+            if logo_uri.is_some() {
+                debug!(
+                    "{}",
+                    format!(
+                        "Downloading client logo from url: {}",
+                        logo_uri.clone().unwrap().as_str()
```

Comment:
Maybe you can use `&logo_uri.unwrap().as_str()` so you don't have to clone

Reply:
Thanks! I think once we have some Rust linting in place using clippy, such issues would be reported during builds as well 👍

Reply:
Yes  clippy check is a good idea as a workflow addition